src/App.tsx

```diff
@@ -0,0 +1,11 @@
+import { Outlet } from 'react-router-dom';
+import Main from './pages/Main/Main';
+
+export default function App() {
+  return (
+    <div style={{ display: 'flex' }}>
+      <Main />
+      <Outlet />
```

Comment:
It's better to put `Main` page into the router config itself. 

In real-life projects having some of the Pages in Router config and some of them in rendering functions, leads developers to questions like "Why do I see this page rendered", and "Where is this page coming from"

*Better to make something like this*:
```tsx
const Main = () => {
return (
    <div>
       <Outlet/>
    <div>
);
}
```

and reconfigure router config main to have children

Reply:
fixed;)